ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -214,6 +252,7 @@ StatusModal {
             StatusColorSelectorGrid {
                 id: colorSelectionGrid
                 anchors.horizontalCenter: parent.horizontalCenter
+                enabled: accountNameInput.valid
                 titleText: qsTr("color").toUpperCase()
```

Comment:
:D

Reply:
@caybro hahahaha that's the best practice thing ;)